packages/csdp/csdp.6.2.0/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+maintainer: "Nikolay Khachatryan <i20.khachatryan.nikolay@etud.ufar.am>"
```

Comment:
@nlaykh Could you please use another email address? Given I guess this one is temporary

Reply:
@erikmd This email is not temporary